pkg/kv/kvserver/replica_raft.go

```diff
@@ -667,7 +666,7 @@ var noSnap IncomingSnapshot
 // non-sensitive cue as to what happened.
 func (r *Replica) handleRaftReady(
 	ctx context.Context, inSnap IncomingSnapshot,
-) (handleRaftReadyStats, string, error) {
```

Comment:
Yay for getting rid of this `string`, I was recently wondering why it had been there.